scripts/getFilesInFolder.js

```diff
@@ -0,0 +1,18 @@
+var fs = require("fs");
+
+// Utility function that collects the stats of every file in a directory
+// as well as in its subdirectories.
+function getFilesInFolder(folderName, fileList) {
+    var folderFiles = fs.readdirSync(folderName);
+    folderFiles.forEach(function(file) {
+        var fileStats = fs.statSync(folderName + "/" + file);
```

Comment:
Would it be better to use `path.join` here and the two places below? To fully accommodate this being run on Windows/OS X/Linux?